tracer/src/Datadog.Trace/Ci/Tagging/TestModuleSpanTags.cs

```diff
@@ -55,4 +58,23 @@ public TestModuleSpanTags(TestSessionSpanTags sessionTags)
 
     [Tag(CommonTags.OSVersion)]
     public string OSVersion { get; set; }
+
+    [Metric(TestTags.IntelligentTestRunnerSkippingCount)]
+    public double? IntelligentTestRunnerSkippingCount
```

Comment:
Should be an integral number, perhaps?

Reply:
Yes, this is a limitation of the numeric tags in .NET all of them are `doubles` so internally I have an `int` counter but then it gets converted here.